### ISSUE TITLE
incorrect images/IIIF for Wiesbaden

### ISSUE THREAD

**Issue by annamorphism** [DDMAL/cantus#518]
Wiesbaden , D-WI1 2 gives images of only ff. 328-434 (seemingly labeled as images 1-214 in the manifest). 
The chant part of the manuscript is over at 466r--apparently not included in the manifest (?).

**Comment by dchiller** [comment]
Yeah, it seems that the library where we get the manifest from only has a smaller number of folios included? 

There is another library that has the whole thing (https://hlbrm.digitale-sammlungen.hebis.de/handschriften-hlbrm/content/pageview/450566), but it doesn't seem like they use a IIIF api? I will continue searching...

**Comment by annamorphism** [comment]
yeah, these are the images CantusDB uses and I couldn't find IIIF information for them (but I hoped maybe I just hadn't found it yet...). You'd think the library where it actually lives  would have the best images...
Anyway I would put this back in the pile, because images without chants aren't really useful for Cantus Ultimus. If it turns out they are out there somewhere (like the folios we have now) we can deal with it then...
